docs/automated-testing/getting-started.md

```diff
@@ -0,0 +1,125 @@
+Select **Add package from git URL** and paste in the following URL:
+
+```
+https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing#v0.0.1
+```
+
+* **NOTE**: The package uses [TextMeshPro](https://docs.unity3d.com/Manual/com.unity.textmeshpro.html) to provide UI elements. If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+* **NOTE**: The package uses [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) to serialize state for recordings.
+
+![The Unity Package Manager dialog with the "Add package from git URL" option selected.](img/getting-started/add-package-from-git.png)
+
+## Adding the Automation Controller and components to your scene
+
+Once the package is installed, you need to add an Automation Controller to the scene you wish to automate.
+The Automation Controller is the hub for all automation in a scene.
+Bots are spawned as children of the Automation Controller,
+and Entity Discoverers (see below) should be added as children of the Automation Controller.
+The Controller also provides APIs for spawning bots and managing recordings (via a separate, Automation Recorder component attached to the same object).
+
+To add the Automation Controller to your scene, click **GameObject > Regression Games > Automation Controller** in the Unity menu.
+
+![The "Automation Controller" option in the Regression Games submenu of the GameObject menu](img/getting-started/add-automation-controller-menu.png)
+
+**By default**, the Automation Controller and all it's children will mark themselves as [DontDestroyOnLoad](https://docs.unity3d.com/ScriptReference/Object.DontDestroyOnLoad.html). This ensures that bots continue to run when you load a new scene in your game. You can disable this behavior by unchecking the "Dont Destroy On Load" checkbox on the Automation Controller component.
+
+![The Automation Controller component with the "Dont Destroy On Load" checkbox checked](img/getting-started/ac-dont-destroy-on-load.png)
+
+## Adding Entity Discoverers to your scene
+
+In order to automate entities in your game, the Automation Controller needs to be able to find them.
+The Automation Controller uses Entity Discoverers to find entities in the scene that can be automated.
+The package provides a few built-in Entity Discoverers, but you can also create your own.
+To add one of the built-in discoverers, open the **GameObject > Regression Games > Discovery** menu and select the discoverer you want to add:
+
+![The "UI Element Discoverer" option in the Regression Games submenu of the GameObject menu](img/getting-started/add-uielement-discoverer.png)
+
+As long as you have an Automation Controller in your scene, the Entity Discoverer will be **automatically** be added as a child of the Automation Controller, no matter what object is selected in the Hierarchy window.
```

Comment:
I submitted an additional comment in my review but it didn't make it into the review?? Reposting...

I can't seem to add an Entity Discovery even after adding my Automation Controller. I get an error message about there being no Automation Controller in the scene.

https://www.loom.com/share/eea0cf42fb4a4da9bf3769b5ed7052ba?sid=42885d93-71c7-4527-9736-13a595c6170c

Reply:
Looks like the scripts aren't properly loaded (you can see the missing script warnings in the editor). That would cause us to fail to recognize the Automation Controller. Now, the bigger question is why those scripts weren't loaded 🤔

Reply:
If you have a chance, could you check the Console for any compilation errors? I notice you're doing this in RGBossRoom, which already has the existing RG SDK installed. It's possible there's a conflict between them. I haven't tested the two together yet (I intended to do so, but since we're focusing on new customers it wasn't the top priority).

Reply:
I'm not seeing any console errors. I tried removing the package, adding it back to the project, and adding a new Automation Controller and the console gives me nada. It looks like the prefab is missing scripts when the package is added.

Reply:
This should be fixed by https://github.com/Regression-Games/RegressionGames.Unity.Testing/pull/8 . Feel free to wait for that or use the branch directly to unblock your testing!